src/ext/java/org/opentripplanner/ext/geocoder/LuceneIndex.java

```diff
@@ -194,7 +177,7 @@ public Stream<StreetVertex> queryStreetVertices(String query, boolean autocomple
    *    one of those is chosen at random and returned.
    */
   public Stream<StopCluster> queryStopClusters(String query) {
-    return matchingDocuments(StopCluster.class, query, true).map(LuceneIndex::toStopCluster);
+    return matchingDocuments(StopCluster.class, query, false).map(LuceneIndex::toStopCluster);
```

Comment:
Why did the `autocomplete` parameter change? Would it make sense to expose it in the API?

Reply:
Lucene's suggest search was not flexible enough for all the use cases that were requested as it lacks the powerful query builder capabilities of the regular, non-suggest search.

For the stop cluster use case we also don't need to be able to switch between the two, since we have pretty a pretty narrow use case.

Does this make sense?